README.md

```diff
@@ -1,4 +1,174 @@
-### Running notebooks
+## API
+
+#### convert_from_process_graph
+
+```python
+convert_from_process_graph(
+    process_graph,
+    n_output_bands=1,
+    sample_type="FLOAT32",
+    units=None,
+    bands_dimension_name="bands",
+    temporal_dimension_name="t"
+)
+```
+
+###### Parameters
+
+* **`process_graph`**: *dict*
+
+    Process graph JSON as Python dict object.
+
+* **`n_output_bands`**: *int, optional*. Default: `1`
+
+    Number of output bands in the evalscript. This can be set if the value is known beforehand. See [docs](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#input-object-properties).
```

Comment:
Shouldn't the URL be for the output bands not input bands?

https://docs.sentinel-hub.com/api/latest/evalscript/v3/#output-object-properties

Reply:
Should be fixed, as well as the rest.

Reply:
Looks good.